ethereum/kycdao-ntnft/hardhat.config.ts

```diff
@@ -120,6 +138,22 @@ const config: HardhatUserConfig = {
       },
       chainId: 421613,
     },
+    baseTestnet: {
+      url: 'https://goerli.base.org',
+      accounts: {
+        mnemonic: test_mnemonic(),
+      },
+      chainId: 84531,
+    },
+    zkTestnet: {
+      url: "https://testnet.era.zksync.dev", // The testnet RPC URL of zkSync Era network.
+      accounts: {
+        mnemonic: test_mnemonic(),
+      },      
+      ethNetwork: "goerli", // The Ethereum Web3 RPC URL, or the identifier of the network (e.g. `mainnet` or `goerli`)
```

Comment:
why does this one have `ethNetwork` and Base doesn't, while Base has chainID and this one doesn't?